lib/config.js

```diff
@@ -79,7 +75,7 @@ config.mongodb.writeOptions = {
 
 config.mongodb.requirements = {};
 // server version requirement with server-style string
-config.mongodb.requirements.serverVersion = '>=4.2';
+config.mongodb.requirements.serverVersion = '>=5';
```

Comment:
The docs and tests are using v6.  Is there a reason to have this lower?

Reply:
Yes, there is software out there still using 5 for which this works -- we can bump it further in the future to avoid additional blockers on getting this update through.